R/session_run_hooks_custom.R

```diff
@@ -79,9 +79,9 @@ hook_view_metrics <- function(props, mode_key) {
 hook_progress_bar <- function(label, steps) {
   
   format <- if (is.null(steps))
-    paste("[:spin]", label, "-- loss: :loss, step: :step")
+    paste("[:spin]", label, "-- loss: :loss, step: :step \n")
```

Comment:
Why are you adding a newline here? The progress bar needs to redraw the line as new iterations are run; wouldn't this break that (you'd have a new line for each progress redraw)?

Reply:
That is for some testing! Will remove! Thanks for spotting it!